Fix interpreter debug logging message

cccclai opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63499

https://github.com/pytorch/pytorch/pull/62418 combine the instruction and debug handle. This change fix the debugging message.

Differential Revision: [D30390699](https://our.internmc.facebook.com/intern/diff/D30390699/)

Commit:
Fix interpreter debug logging message

https://github.com/pytorch/pytorch/pull/62418 combine the instruction and debug handle. This change fix the debugging message.

Differential Revision: [D30390699](https://our.internmc.facebook.com/intern/diff/D30390699/)

[ghstack-poisoned]

kimishpatel reviewed (approved):
Looks good. Thanks for fixing this.

raziel reviewed (approved):
thanks

Commit:
Update on "Fix interpreter debug logging message"


https://github.com/pytorch/pytorch/pull/62418 combine the instruction and debug handle. This change fix the debugging message.

Differential Revision: [D30390699](https://our.internmc.facebook.com/intern/diff/D30390699/)

[ghstack-poisoned]

Commit:
Update on "Fix interpreter debug logging message"


https://github.com/pytorch/pytorch/pull/62418 combine the instruction and debug handle. This change fix the debugging message.

Differential Revision: [D30390699](https://our.internmc.facebook.com/intern/diff/D30390699/)

[ghstack-poisoned]

Commit:
Update on "Fix interpreter debug logging message"


https://github.com/pytorch/pytorch/pull/62418 combine the instruction and debug handle. This change fix the debugging message.

Differential Revision: [D30390699](https://our.internmc.facebook.com/intern/diff/D30390699/)

[ghstack-poisoned]
